Game start popup reshuffle

comma opened the pull request:
Adds FUCKING LORE BLURB there maybe people will ACTUALLY READ IT.
Adds name of character you have currently selected
Changes layout a bit to accomodate fat text blob.

![](https://i.imgur.com/T6zTXaS.png)

comma committed:
Game start popup reshuffle

Adds FUCKING LORE BLURB there maybe people will ACTUALLY READ IT.
Adds name of character you have currently selected
Changes layout a bit to accomodate fat text blob.

afterthought2 commented:
You double-checked that it looks sane with the alt stylesheet? There's a prefs option to make all fake nanoui windows look awful, and it applies to this one too.

xales commented:
Might be worth throwing in `"ship"` or `"starship"` or something somewhere instead of one of the `"vessel"`s so people might be capable of understanding the setting a bit more... 🙄

comma commented:
>You double-checked that it looks sane with the alt stylesheet? 

Didn't know that exist, will check

And sure

comma commented:
Added spaces between links/buttons, aside from that ugly style works fine.
Replaced vessel with starship in first sentence
Also welcome now uses loaded char's name for immulsions, and shows what high pref job (if any) you are set up as

![](https://i.imgur.com/vFhjorJ.png)
